plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/semantics/context/SQLQueryCombinedContext.java

```diff
@@ -43,14 +44,14 @@ public SQLQueryRowsSourceModel findRealSource(@NotNull DBSEntity table) {
 
     @NotNull
     @Override
-    public DBSEntity findRealTable(@NotNull List<String> tableName) {
-        return anyOfTwo(parent.findRealTable(tableName), otherParent.findRealTable(tableName)); // TODO consider ambiguity
+    public DBSEntity findRealTable(List<String> tableName, DBRProgressMonitor monitor) {

```

Comment:
No @NotNull annotation anymore for tableName?

Reply:
thank you